packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -927,20 +944,26 @@ class RenderListWheelViewport
     Matrix4 cylindricalTransform,
     Offset offsetToCenter,
   ) {
+    // Paint child with cylindrically.
```

Comment:
Nit: Should this just be "Paint child cylindrically."?  Or does it mean to paint it with the "cylindrically" method?  Also see line 956 below.

Reply:
Forgot to delete it I guess.